apps/game-service/src/main.ts

```diff
@@ -250,32 +253,22 @@ const ioc = new InjectionContainer()
   .provide(ctxLogger, logger)
   .provide(ctxActorModelLookup, actorModels)
   .provide(ctxRng, rng)
-  .provide(ctxNpcSpawner, npcSpawner)
   .provide(ctxGameEventClient, gameEventBroadcastClient)
-  .provide(
-    ctxItemDefinitionLookup,
-    createItemDefinitionLookup(
-      await promiseFromResult(db.selectAllItemDefinitions()),
-    ),
-  )
+  .provide(ctxNpcSpawner, npcSpawner)
+  .provide(ctxItemDefinitionLookup, itemDefinitionLookup)
   .provide(ctxDbSyncSession, dbSyncSession);
 
-logger.info(`Getting all NPC rewards...`);
-const npcRewardSystem = new NpcRewardSystem(
-  ioc,
-  await promiseFromResult(db.selectAllNpcRewards(area.id)),
-);
-
 const npcAi = new NpcAi(gameState, gameStateServer, area, rng);
 
+// Start all functionality immediately (movement, NPCs, combat, state sync)
```

Comment:
remove these comments, stop adding comments in this PR. Your comments are unnecessary